### PR TITLE
Forms Component: login and signup page

### PR DESCRIPTION
**Pull request by tushartiwari7**
Two types of forms included

- Login Form
- Signup form
login
signup

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
Looks good

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```
Comment (rkapoor10) [inline .vscode/settings.json]: maybe this should be in .gitignore

File: components/forms/main.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                 class="input full-width px-sm py-xs my-xs rounded-s"
+           />
+        </div>
+        <div class=" m-xs">
+          <label class="fs-s">
+            Password 
+          </label>
+          <input 
+                 type="password" 
+                 placeholder="********"
```
Comment (rkapoor10) [inline components/forms/main.html:27]: you can use placeholders to give a message like - "Enter your password" just a suggesstion.

**Review by rkapoor10 — APPROVED** [review]
Code is well structured; good to merge.

File: .vscode/settings.json (hunk 1 above)
Comment (tushartiwari7) [reply .vscode/settings.json]: done

File: components/forms/main.html (hunk 2 above)
Comment (tushartiwari7) [reply components/forms/main.html:27]: Thanks for the suggestion @rkapoor10, I appreciate that.
But in the Figma template, it was like this only so I am using this.

Comment (tushartiwari7) [reply components/forms/main.html:27]: Done

**Pull request merged**